src/app/pages/5_Find_Assay_Target_Area.py

```diff
@@ -82,15 +82,24 @@
             help="If no value is entered, default value of 20 will be used as the target region slide size",
         )
 
-        max_difference = st.number_input(
-            label="Enter the maximum differences allowed between target area and target sequences",
+        max_differenc_o = st.number_input(
+            label="Enter the maximum differences allowed between primer and target sequences",
             min_value=0,
             value=5,
             step=1,
             help="If no value is entered, default value of 5 will be used as the maximum differences allowed between "
             "target area and target sequences",
         )
 
+        max_difference_ot = st.number_input(
+            label="Enter the maximum differences allowed between primer and off-target sequences",
+            min_value=0,
```

Comment:
[question] should be the min_value be 0 here? Would it make sense to have an off-target with 0 differences?